ultravox/training/train.py

```diff
@@ -155,6 +159,8 @@ def main() -> None:
                 )
 
     model.print_trainable_parameters()
+    # loss_config needs to be passed separately just for model training
+    model.set_loss_config(args.loss_config)
```

Comment:
suggest moving this before line 128 so all model init happens in one place. Also, suggest only calling `set_loss_config` if `args.loss_config` isn't None, to avoid the need for `Optional`

Reply:
Done. args.loss_config is updated to always have the LossConfig type.